app/models/service_offering.rb

```diff
@@ -1,6 +1,6 @@
 class ServiceOffering < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id", :inverse_of => :service_offerings
-  belongs_to :service_broker
```

Comment:
Oops how did that slip in

Reply:
it was added by a group of evil hobgoblins :-)